catalog/spatial/registry/registry-policy-plugin/src/main/java/org/codice/ddf/registry/policy/RegistryPolicyPlugin.java

```diff
@@ -99,30 +97,19 @@ public void setRegistryDisabled(boolean registryDisabled) {
         this.registryDisabled = registryDisabled;
     }
 
-    /**
-     * Getter for the permissions string array
-     *
-     * @return The list of bypass permissions
-     */
-    public List<String> getRegistryBypassPolicyStrings() {
-        return this.registryBypassPolicyStrings;
+    public void setCreateAccessPolicyStrings(List<String> accessPolicyStrings) {
+        parsePermissionsFromString(accessPolicyStrings, createAccessPolicy);
     }
 
-    public List<String> getWriteAccessPolicyStrings() {
-        return writeAccessPolicyStrings;
+    public void setUpdateAccessPolicyStrings(List<String> accessPolicyStrings) {
+        parsePermissionsFromString(accessPolicyStrings, updateAccessPolicy);
     }
 
-    public void setWriteAccessPolicyStrings(List<String> writeAccessPolicyStrings) {
-        this.writeAccessPolicyStrings = writeAccessPolicyStrings;
-        parsePermissionsFromString(writeAccessPolicyStrings, writeAccessPolicy);
-    }
-
-    public List<String> getReadAccessPolicyStrings() {
-        return readAccessPolicyStrings;
+    public void setDeleteAccessPolicyStrings(List<String> accessPolicyStrings) {
```

Comment:
deleteAccessPolicy is declared as an instance variable, why do you need to pass it in as an argument?